examples/polkadot/compile.sh

```
@@ -1,88 +1,354 @@
+  echo "$reason"
+}
+
+# ----------------------------------------------------------------------------
+# Main
+# ----------------------------------------------------------------------------
+
+build_compiler
+
+# Per-runtime results. Each entry is a tab-separated row used to render the README.
+declare -a RESULTS
+RESULTS_HEADER="name|compressed|wasm|imports|float_ops|status|jam|reason|elapsed"
```

Shellcheck (0.11.0)

[warning] RESULTS_HEADER appears unused. Verify use (or export if used externally).

(SC2034)